tests/commands/test_runner.py

```diff
@@ -147,3 +147,26 @@ def test_patch_modules_from_env(self):
         update_patched_modules()
         assert EXTRA_PATCHED_MODULES["boto"] == True
         assert EXTRA_PATCHED_MODULES["django"] == False
+
+    def test_sitecustomize_run(self):
```

Comment:
Without the patch, this test fails:
```
  File "tests/commands/ddtrace_run_sitecustomize.py", line 9, in <module>
    ok_(sitecustomize.CORRECT_IMPORT)
AttributeError: module 'sitecustomize' has no attribute 'CORRECT_IMPORT'
```

Even if we don't test `sitecustomize.py` automatic call (we can assume it works if the `PYTHONPATH` is properly set), we test that our `bootstrap/sitecustomize.py` isn't in the `PYTHONPATH` once it has been loaded. In that case, when the built-in `site.py` is going to import that module, the one defined by the user will be found first.